src/udiskslinuxblock.c

```diff
@@ -1530,14 +1530,16 @@ has_whitespace (const gchar *s)
 }
 
 static gchar *
-make_block_luksname (UDisksBlock *block)
+make_block_luksname (UDisksBlock *block, GError **error)
 {
-  const gchar *uuid = udisks_block_get_id_uuid (block);
-
-  if (uuid && *uuid)
-    return g_strdup_printf ("luks-%s", uuid);
+  gchar *uuid = bd_crypto_luks_uuid (udisks_block_get_device (block), error);
 
-  return NULL;
+  if (!uuid)
+    return NULL;
+  else
+    {
+      return g_strdup_printf ("luks-%s", uuid);
+    }
```

Comment:
Extra braces here. And maybe switching the branches here (together with the condition of course) would make more sense?